Sources/Parsing/Builders/OneOfBuilder.swift

```diff
@@ -1,7 +1,21 @@
-/// A custom parameter attribute that constructs a parser that attempts to run a number of parsers,
-/// one after the other, till one succeeds with an output.
+/// A custom parameter attribute that constructs parsers from closures. The constructed parser
+/// runs each parser in the closure, one after another, till one succeeds with an input.
```

Comment:
Took some of this wording from the `@ViewBuilder` documentation.

The "custom parameter attribute" is weird to me. Kinda feel like we could say something similar...

Reply:
Do you mean say something other than "custom parameter attribute"?